src/main/java/se/sundsvall/seabloader/scheduler/dbcleaner/DatabaseCleanerSchedulerService.java

```diff
@@ -35,11 +38,18 @@ public void execute() {
 		final var entitiesToRemove = invoiceRepository.countByStatusIn(STATUSES_TO_REMOVE);
 		if (entitiesToRemove > 0) {
 			LOGGER.info(LOG_ENTITIES_REMOVAL, entitiesToRemove, STATUSES_TO_REMOVE);
```

Comment:
STATUS_OF_ENTITIES_TO_REMOVE
STATUS_FOR_REMOVAL

(since the job are not removing the status.)